bittensor/_metagraph/metagraph_impl.py

```diff
@@ -319,33 +306,34 @@ async def _async_sync( self, force: bool = False ):
             self.neurons.append( torch.nn.Parameter( torch.tensor([], dtype=torch.int64), requires_grad=False ) )
 
         # Fill pending queries.
-        pending_queries = []
-        for uid, lastemit in chain_lastemit.items():
-            if lastemit > old_block or force == True:
-                pending_queries.append((False, uid))
-
-        # Fill buffers with retry.
-        # Below fills buffers for pending queries upto the rety cutoff.
-        retries = 0
-        max_retries = 3
-        while True:
-            if retries >= max_retries:
-                logger.critical('Failed to sync metagraph. Check your subtensor connection.')
-                raise RuntimeError('Failed to sync metagraph. Check your subtensor connection.')            
-            queries = []
-            for code, uid in pending_queries:
-                if code == False:
-                    queries.append( self.fill_uid( uid = uid ) )
-            if len(queries) == 0:
-                # Success
-                break
-            pending_queries = [await query for query in tqdm.asyncio.tqdm.as_completed( queries )]
-            retries += 1 
+        # pending_queries = []
+        # for uid, lastemit in chain_lastemit.items():
+        #     if lastemit > old_block or force == True:
+        #         pending_queries.append((False, uid))
+
+        # # Fill buffers with retry.
+        # # Below fills buffers for pending queries upto the rety cutoff.
+        # import pdb; pdb.set_trace()
+        # retries = 0
+        # max_retries = 3
+        # while True:
+        #     if retries >= max_retries:
+        #         logger.critical('Failed to sync metagraph. Check your subtensor connection.')
+        #         raise RuntimeError('Failed to sync metagraph. Check your subtensor connection.')            
+        #     queries = []
+        #     for code, uid in pending_queries:
+        #         if code == False:
+        #             queries.append( self.fill_uid( uid = uid ) )
+        #     if len(queries) == 0:
+        #         # Success
+        #         break
+        #     pending_queries = [query for query in queries]
+        #     retries += 1 
```

Comment:
@unconst i'm not sure if i understood this logic quite correctly. It seems the issue is that we used to use [await query for query in tqdm.asyncio.tqdm.as_completed( queries )]. My understanding is this logic waits for the queries to complete. But since we're no longer using asyncio i'm not sure if we need all this logic anymore. Do let me know your thoughts!